docs/sources/mimir/operators-guide/get-started/_index.md

```diff
@@ -9,24 +9,25 @@ weight: 10
 
 # Get started with Grafana Mimir
 
-There are two different options for getting started with Grafana Mimir:
+You can get started with Grafana Mimir _imperatively_ or _declaratively_:
 
-- The written tutorial provides a series of imperative commands to start a single Mimir process.
-- The visual tutorial (in the form of a video) uses `docker-compose` to declaratively deploy multiple Mimir processes.
+- **Imperatively**: The written instructions that follow contain commands to help you start a single Mimir process. You would need to perform the commands again to start another Mimir process.<p>
```

Comment:
The `<p>` at the end doesn't look correct.

Reply:
Thanks for catching that. Correct, it no longer makes sense because I moved some content around. Will open a quick PR.